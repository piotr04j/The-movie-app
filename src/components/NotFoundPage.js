import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => {
    return (
        <div className="container my-3">
            <p className="font-weight-bold">Page not found</p>
            <p>Back to home page:</p>
            <Link to='/' className="btn btn-primary">Home Page</Link>
        </div>
    )
};

export default NotFoundPage;
