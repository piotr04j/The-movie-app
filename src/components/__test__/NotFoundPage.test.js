import React from 'react';
import renderer from 'react-test-renderer';
import NotFoundPage from '../NotFoundPage';
import { MemoryRouter } from 'react-router-dom';


it('renders correctly', () => {
    const tree = renderer.create(<MemoryRouter><NotFoundPage /></MemoryRouter>).toJSON();
    expect(tree).toMatchSnapshot();
});