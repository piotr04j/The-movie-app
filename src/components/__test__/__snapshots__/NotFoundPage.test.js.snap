// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container my-3"
>
  <p
    className="font-weight-bold"
  >
    Page not found
  </p>
  <p>
    Back to home page:
  </p>
  <a
    className="btn btn-primary"
    href="/"
    onClick={[Function]}
  >
    Home Page
  </a>
</div>
`;
